core/src/main/scala/caliban/introspection/adt/__Type.scala

```diff
@@ -10,4 +10,19 @@ case class __Type(
   enumValues: __DeprecatedArgs => Option[List[__EnumValue]] = _ => None,
   inputFields: Option[List[__InputValue]] = None,
   ofType: Option[__Type] = None
-)
+) {
+  def |+|(that: __Type): __Type = __Type(
```

Comment:
This too would be called `<>` in ZIO (essentially `<+>` but without preserving the distinction between the sides). Not sure if it's a good idea here or if it's better to go with Semigroup-inspired `|+|`.